prow/hook/deployment.yaml

```diff
@@ -23,7 +23,7 @@ spec:
       terminationGracePeriodSeconds: 180
       containers:
       - name: hook
-        image: gcr.io/k8s-prow/hook:v20190307-bbad344
+        image: gcr.io/k8s-prow/hook:v20190402-d65ae756e
```

Comment:
Same as above regarding is this worth it

Reply:
I intended this line to be a placeholder to update hook's image once https://github.com/kubernetes/test-infra/pull/12011 had been merged (it now is), enabling an in-built probe - as is the case with deck in this PR.